pfb-analysis/connectivity/access_trails.sql

```diff
@@ -0,0 +1,49 @@
+----------------------------------------
+-- INPUTS
+-- location: neighborhood
```

Comment:
Added in https://github.com/azavea/pfb-network-connectivity/pull/153/commits/c185b0d4c98c095c6d610babf29139ca19119f8f